File: datasets/IR-Plag/convertai-original-java-rest-kotlin/case-04/plagiarized/L4/01/L4.kt
package IR

fun print(mil: Int) {
    var mil = mil
    while (mil <= 10) {
        println("$mil\t\t${mil * 1.609}")
        mil++
    }
}

fun main(args: Array<String>) {
    var mil = 1
    println("Miles\t\tKilometers")
    println("-------------------------------")
    print(mil)
}

File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-04/plagiarized/L4/01/L4.ts
package IR;

class L4 {
    static print(mil: number): void {
        while (mil <= 10) {
            console.log(`${mil}\t\t${mil * 1.609}`);
            mil++;
        }
    }

    static main(args: string[]): void {
        let mil: number = 1;
        console.log("Miles\t\tKilometers");
        console.log("-------------------------------");
        this.print(mil);
    }
}

